xbmc/guilib/GUIControl.cpp

```diff
@@ -185,7 +186,15 @@ void CGUIControl::DoRender()
       g_graphicsContext.SetStereoFactor(m_stereo);
 
     GUIPROFILER_RENDER_BEGIN(this);
+
+    if (m_hitColor != 0xFFFFFFFF)
```

Comment:
You can leave it CGUIInfoColor in factory. Just change the member and signature of CGUIControl::SetHitRect to colot_t. MergeAlpha wants color_t anyway.